extensions/cornerstone-3d/src/commandsModule.js

```diff
@@ -19,6 +26,15 @@ const commandsModule = ({ servicesManager }) => {
     getActiveViewportEnabledElement: () => {
       return _getActiveViewportEnabledElement();
     },
+    toggleCine: () => {
+      const { viewports } = ViewportGridService.getState();
+      const { isCineEnabled } = CineService.getState();
+      CineService.setIsCineEnabled(!isCineEnabled);
+      ToolBarService.setButton('Cine', { props: { isActive: !isCineEnabled } });
+      viewports.forEach((_, index) =>
+        CineService.setCine({ id: index, isPlaying: false })
```

Comment:
And then the CineService should just set is playing on the Cornerstone3D layer, rather than it being props?

Its weird that the CINE provider is in OHIF UI anyways. Shouldn't it be in the cornerstone extension?